src/components/LeetcodeText.js

```diff
@@ -0,0 +1,27 @@
+import styles from '../css/LeetcodeText.module.css'
+
+const LeetcodeText = props => {
+  return (
+    <div className={styles.container}>
+      <p className={styles.title}>Leetcode</p>
```

Comment:
I think the marginBottom for styles.title is different for desktop and mobile. With the media hook, you can make conditional styling:

`style={isDesktop ? { marginBottom: '7px' } : { marginBottom: '5px' }}`